### ISSUE TITLE
Is it possible to separate the css file?

### ISSUE THREAD

**Issue by heatherKoo07** [nguyenbathanh/react-loading-screen#20]
Hi, thanks for sharing your approach.
One question about the style block in HTML.
Is it possible to separate the CSS codes in a file like "progress.css" and link it to the HTML file?
For example, `<link rel="stylesheet" href="progress.css">`

**Comment by nguyenbathanh** [comment]
@heatherKoo07 Yes, it's possible. However I only recommend that approach unless your external CSS host has very good performance. Otherwise, you may see flash effect while opening your page until the style sheet get delivered.
